src/test/common/utils/numericUtils.ts

```diff
@@ -48,28 +60,44 @@ describe('toNumberOrUndefined', () => {
       expect(toNumberOrUndefined(-10)).to.be.eq(-10)
     })
 
-    it('negative numeric string given', () => {
+    it('negative numeric given', () => {
```

Comment:
No numeric value is passed to the `toNumberOrUndefined` function so tests should not have numeric world in description. I suggested to use numeric only when numbers are passed in (not strings)